proto/rill/admin/v1/api.proto

```diff
@@ -1184,6 +1184,7 @@ message CreateAssetRequest {
   string name = 3 [(validate.rules).string.min_len = 1]; // Eg. "rilldata__project_name"
   string extension = 4; // Eg. ".tar.gz"
   bool cacheable = 5;
+  int64 estimated_size_bytes = 6;
```

Comment:
Now this needs to be set while uploading project assets as well I don't see that change

Reply:
It is optional – if not set, it will just be enforced during the upload itself (which was the previous behavior). Just added it here to error early with a nicer error message.

It is not quite a trivial refactor for the project upload use cases, so adding a separate backlog issue to make the refactor there :)

Reply:
OK I think its fine then

Reply:
don't think its really necessary to add fail early check